api/spec/src/subscription.tsp

```diff
@@ -0,0 +1,390 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi";
+import "@typespec/openapi3";
+import "@typespec/versioning";
+
+import "./types.tsp";
+import "./errors.tsp";
+import "./prices.tsp";
+import "./ratecards.tsp";
+import "./jsonpatch.tsp";
+import "./discounts.tsp";
+import "./entitlements/main.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.OpenAPI;
+using TypeSpec.Versioning;
+
+namespace OpenMeter;
+
+@route("/api/v1/subscriptions")
+@tag("Subscriptions")
+interface Subscriptions {
+  @get
+  @operationId("getSubscription")
+  @sharedRoute
+  get(
+    @path subscriptionId: ULID,
+    @query expand?: false,
```

Comment:
Should we use a more detailed expansion?

```
expand: ['customer']
```

See https://docs.stripe.com/expand

Reply:
I see little added value, I think the current form is the 90% usecase